packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -109,7 +109,7 @@ export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderS
               // so far. That's the reasons we need to use "+" to show we might have more items than count
               // indicates.
             }
-            <span>({ group.count }{ group.hasMoreData && '+' })</span>
+            <span>{ ' ' }({ group.count }{ group.hasMoreData && '+' })</span>
```

Comment:
The extra space here is not RTL friendly. Instead, just put a class on this, and set the padding to `0 8px`. That will enforce that there is consistent padding between the name and the count.